hack/dind-cluster.sh

```diff
@@ -607,6 +642,17 @@ case "${1:-""}" in
       esac
     done
 
+   # OVN requires CNI network plugin and OVN_ROOT to be set
+   if [[ -n "${USE_OVN}" ]]; then
```

Comment:
With `set -o nounset` these will fail, use `"${USE_OVN:-}"` *etc* for them

Reply:
done